amf-client/shared/src/test/resources/maker/declared-types.json

```diff
@@ -3,9 +3,6 @@
   "info":{
     "title": "test types",
     "description": "empty api only for test types"
-  },
-  "paths": {
```

Comment:
Weird, without this the API is invalid

Reply:
This file is no used to test validation, just to test parsing. If path is present, the we will have an empty field endpoints that cannot be obtained in a raml api.
As the fixture is the same for deferents specs (on purpose) remove the paths key for the test source is the most convenience solution.